app/javascript/components/patient/laboratory/FirstPositiveLaboratory.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Button, Form } from 'react-bootstrap';
+import moment from 'moment';
+
+import confirmDialog from '../../util/ConfirmDialog';
+import LaboratoryModal from './LaboratoryModal';
+
+class FirstPositiveLaboratory extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showModal: false,
+    };
+  }
+
+  handleDelete = async () => {
+    const options = {
+      title: 'Delete lab result',
+      okLabel: 'Delete',
+      cancelLabel: 'Cancel',
+    };
+    if (await confirmDialog('Are you sure you want to delete this lab result?', options)) {
+      this.props.onChange(null);
+    }
+  };
+
+  render() {
+    return (
+      <React.Fragment>
+        {this.props.lab ? (
+          <div className={this.props.displayedLabClass || 'my-2'}>
+            <div className="first-positive-lab-result-header">
+              <Form.Label className="input-label mb-1">FIRST POSITIVE LAB RESULT</Form.Label>
+              <div className="edit-link">
+                <Button variant="link" id="delete-first_positive_lab" className="py-0 px-1 icon-btn-dark" onClick={this.handleDelete}>
```

Comment:
this needs an aria-label

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/a79a65940f2a0383f7f4d8b28330d2b7e1db30ae